draft-ietf-acme-acme.md

```diff
@@ -734,12 +742,12 @@ the following table and whose values are the corresponding URLs.
 
 | Field          | URL in value         |
 |:---------------|:---------------------|
-| newNonce       | New nonce            |
-| newAccount     | New account          |
-| newOrder       | New order            |
-| newAuthz       | New authorization    |
-| revokeCert     | Revoke certificate   |
-| keyChange      | Key change           |
+| newNonce       | new-nonce            |
```

Comment:
I don't think these should be made to match example URLs. To me the table is meant to convey the fields (keys in the directory), and the human description of what endpoint URL is expected there, not an actual URL or reference to an example URL.